### ISSUE TITLE
Feat - Assignee and Reporter should be marked as invited by default

### ISSUE THREAD

**Issue by franck-boullier** [unee-t/frontend#106]
In current version of PR #91
- List of possible invitee
  - Assignee and Reported should be marked as invited by default
  - Reporter can never be uninvited to a case
  - Assignee cannot be uninvited to a case: the case has to be assigned to someone else first.

<img width="239" alt="invite_users_issue_with_user_list" src="https://user-images.githubusercontent.com/31331637/35702567-6d985e80-07d4-11e8-95b2-337ad2684403.png">

Maybe we can also highlight the Reporter and Assignee in that list for the avoidance of doubt.

**Comment by nbiton** [comment]
i think this might not be relevant anymore after #73 was implemented. Instead, this should be probably closed, as once #108 is implemented it won't make any more sense.

**Comment by franck-boullier** [comment]
It is not a top priority and could be done at a later stage but it's confusing to see the Reporter and the assignee as possible invitee to the case (see case 61903 for an example of that:
1- Reporter and Assignee are the same: 'Franck - Test user'
![image](https://user-images.githubusercontent.com/31331637/35973808-c283a5de-0d11-11e8-9491-96d4897e7e36.png)

2- 'Franck - Test user' appears on the list of people that can be invited (added in CC) to the case.
![image](https://user-images.githubusercontent.com/31331637/35973715-6da21b72-0d11-11e8-9dbe-fed13c5abf7f.png)

We should either:
Option 1: hide the user from the list of people who can be added in CC if the user is either the reporter or the assignee 
OR
Option 2: add a 'green tick' next to the user to indicate that the user is already involved in the case.

Side not: the color code for the same user is different when:
- User is the same
- User is involved in the case in several capacity, example: user is the Reporter AND the Assignee. This is counter intuitive - putting this here even though I should probably create a separate issue or update issue #49...

**Comment by nbiton** [comment]
We are going for option 1 :)

**Comment by franck-boullier** [comment]
Fixed in the latest version of PR#112
Assignee and Reporters are filtered out from the list of other people who can be invited to the case.
